instrumentation/cassandra/cassandra-4.4/library/src/main/java/io/opentelemetry/instrumentation/cassandra/v4_4/CassandraAttributesExtractor.java

```diff
@@ -36,6 +46,12 @@ public void onEnd(
 
     Node coordinator = executionInfo.getCoordinator();
     if (coordinator != null) {
+      try {
+        updateServerAddressAndPort(attributes, coordinator);
+      } catch (IllegalAccessException e) {
+        logger.error("Error while extracting server address and port", e);
```

Comment:
I'd suggest narrowing this catch down to the one line where it's needed inside of `updateServerAddressAndPort`, and also logging it at `fine`

Reply:
there is no fine.. i logged it at warning level.

Reply:
oh, I see, we use `java.util.logging` in this repo, and this is an slf4j logger